src/components/PoolDashboard/PoolDashboard.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import {useAsync} from 'react-use';
+import { makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import { useQuery } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+import { fromWei } from 'web3-utils'
+import get from 'lodash/get'
+
+import { getCurrentPrize, fetchPoolMetadata } from '../../ethereum/pool';
+import MainButton from '../UI/MainButton';
+
+const GET_POOL = gql`
+  query Pool($poolAddress: String!) {
+    funding(id: $poolAddress) {
+      address,
+      totalStaked,
+      numberOfPlayers
+    }
+  }
+`;
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
```

Comment:
Also, it doesn't show cover images for me. Let me know if you have questions regarding this topic